GPU/Vulkan/TextureCacheVulkan.cpp

```diff
@@ -541,8 +541,10 @@ void TextureCacheVulkan::BuildTexture(TexCacheEntry *const entry) {
 		plan.GetMipSize(i, &mipWidth, &mipHeight);
 
 		int bpp = VkFormatBytesPerPixel(actualFmt);
-		int stride = (mipWidth * bpp + 15) & ~15;  // output stride
-		int uploadSize = stride * mipHeight;
+		int optimalStrideAlignment = std::max(4, (int)vulkan->GetPhysicalDeviceProperties().properties.limits.optimalBufferCopyRowPitchAlignment);
```

Comment:
This isn't actually used?  Shouldn't we have `int byteStride = std::max(optimalStrideAlignment, RoundUpToPowerOf2(mipWidth * bpp));` or something (more complex if it wouldn't always be a power of two)?

-[Unknown]

Reply:
Ugh, it was supposed to be passed in as the second parameter in RoundUpToPower2 (there's now a verison that takes that, which is useful sometimes).